deploy/deploy-market.ts

```diff
@@ -34,6 +34,9 @@ deploy.dependencies = [
   'BridgedPolygonNORI',
   'RestrictedNORI',
 ];
+if (hre.network.name !== 'polygon') {
+  deploy.dependencies = [...deploy.dependencies, 'NoriUSDC'];
```

Comment:
@scottynomad is this still needed?

Reply:
It's needed for local only now but should get updated to exclude mumbai as well.

Reply:
gotcha. I think we can drop this entirely in the future actually by just using a fork of mumbai as the base block in tests

Reply:
yeah using account impersonation